### PR TITLE
Airlock manager can use user resources

### LINKED ISSUE #2499: Airlock Manager can create user resources
## Description

As an Airlock Manager
I want to have a permission to create VMs in workspaces I'm added as an Airlock Manager
So that I can create a VM and use it review an Airlock request

## Acceptance criteria

- [ ] Airlock Manager can create user resources

### PR DESCRIPTION
**Pull request by tanya-borisova**
# Resolves #2499 

## What is being addressed

Allowing Airlock Manager to create and operate user resources.
Airlock Manager needs to have this capacity to be able to create VMs to perform Airlock Requests from. 

## How is this addressed

- Add Airlock Manager to the list of allowed roles for all methods on user resources
- Add Airlock Manager to the list of allowed roles for GET for workspace services and workspace service operations (otherwise, on the UI Airlock manager is going to see errors)
- Restrict Research VM templates to only be used by WorkspaceOwner and WorkspaceResearcher
- Made a change on the UI to allow Airlock Manager to see Connect button
- Made a change in Guacamole to allow Airlock Manager to connect to VMs

### CODE REVIEW

File: templates/workspace_services/guacamole/guacamole-server/guacamole-auth-azure/src/main/java/org/apache/guacamole/auth/azuretre/AuthenticationProviderService.java [hunk 1]
```diff
@@ -67,7 +67,8 @@ public void validateToken(final String token, final UrlJwkProvider jwkProvider)
 
             List<String> rolesList = roles.asList(String.class);
             if (rolesList.stream().noneMatch(x -> x.equalsIgnoreCase("WorkspaceOwner")
-                || x.equalsIgnoreCase("WorkspaceResearcher"))) {
+                || x.equalsIgnoreCase("WorkspaceResearcher")
+                || x.equalsIgnoreCase("AirlockManager"))) {
                 throw new GuacamoleInvalidCredentialsException(
                     "User must have a workspace owner or workspace researcher role", CredentialsInfo.USERNAME_PASSWORD);
```
Comment (damoodamoo) [inline templates/workspace_services/guacamole/guacamole-server/guacamole-auth-azure/src/main/java/org/apache/guacamole/auth/azuretre/AuthenticationProviderService.java:73]: "...or Airlock Manager"

**Review by damoodamoo — APPROVED** [review]
LGTM!

**Comment by tanya-borisova** [comment]
/test-force-approve

No point in running tests as they don't use the new template

**Comment by tanya-borisova** [comment]
^ wrong PR! I do need to run tests on here

**Pull request merged**